tests/test_ecr.py

```diff
@@ -288,3 +288,64 @@ def test_ecr_remove_named(self):
 
     def test_ecr_set_lifecycle(self):
         pass
+
+    def test_ecr_cel_cross_account(self):
+        session_factory = self.replay_flight_data("test_ecr_remove_matched")
+        client = session_factory().client("ecr")
+        name = "test-cel-cross-account"
+        self.create_repository(client, name)
+        client.set_repository_policy(
```

Comment:
Do you need to do this here in the unit test? Can you run this locally and then just filter out to retrieve the intended ECR resource? Simplifies this unit test down a bit